### ISSUE TITLE
(Rubric) Accessibility

### ISSUE THREAD

**Issue by sljohnson32** [sljohnson32/2DoBox-Pivot#21]
4 - The application demonstrates an exceptional knowledge of web accessibility. The application has no aXe-core violations, is fully functional without using a mouse, and the application uses HTML/CSS to ‘chunk’ content into more manageable pieces. The application utilizes semantic HTML.

3 - The application has a strong accessibility presence. The developer showed a strong effort to keep accessibility in mind from the beginning of the project. The application has 0-3 aXe-core violations.
